src/components/Contacts/ContactDetails/ContactDetailsTab/Mailing/ContactDetailsTabMailing.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box } from '@material-ui/core';
+import React, { ReactElement } from 'react';
+import { gqlMock } from '../../../../../../__tests__/util/graphqlMocking';
+import {
+  ContactDetailsTabDocument,
+  ContactDetailsTabQuery,
+  ContactDetailsTabQueryVariables,
+} from '../ContactDetailsTab.generated';
+import { ContactDetailsTabMailing } from './ContactDetailsTabMailing';
+
+export default {
+  title: 'Contacts/Tab/ContactDetailsTab/Mailing',
+};
+
+export const Default = (): ReactElement => {
```

Comment:
Like my question below, will this ever need a loading indicator? If so we could probably add a story here for loading.

Reply:
@TheNoodleMoose I don't think it will need a loading because it is passing in the query results from the ContactDetailsTab.  That view already has a loading screen.